cos-gpu-installer-docker/entrypoint.sh

```diff
@@ -427,28 +427,62 @@ update_host_ld_cache() {
   ldconfig -r "${ROOT_MOUNT_DIR}"
 }
 
+usage() {
+  echo "usage: $0 [-p|-s]"
+  echo "Default behavior installs all components needed for the Nvidia driver."
+  echo "  -p: Install cross toolchain package only"
+  echo "  -s: Skip cross toolchain package installation"
```

Comment:
Why not simply add a check in `install_cross_toolchain_pkg()` function to see if the toolchain already exists and be a no-op if it exists?
Same can be done for `download_kernel_src` too.

Reply:
+1 with what Aditya said. The -s flag shouldn't be necessary. And the -p flag should install both toolchain and kernel source. It helps to decrease the latency even if pre-compiled driver is not available.

Reply:
Thanks for pointing this out. Moved the container path to /build/cos-tools in order to check if toolchain has been installed. 

`download_kernel_src` already checks if it is previously available.